powersimdata/design/transmission/tests/test_upgrade.py

```diff
@@ -139,43 +139,43 @@ def setUp(self):
         self.grid.id2zone = {201: "Wahington", 202: "Oregon"}
         self.grid.zone2id = {"Washington": 201, "Oregon": 202}
 
-    def test_internal_W(self):
+    def test_internal_W(self):  # noqa: N802
```

Comment:
These function names are an artifact of old tests where we had dummy zones `"E"` and `"W"`. Now that we've changed the tests to `Oregon`/`Washington`, it probably makes sense to change the test names.

Reply:
I gave it a shot, let me know if it makes sense.